### ISSUE TITLE
Missing Study

### ISSUE THREAD

**Issue by LauraMTG** [clarity-h2020/data-package#79]
Hi @patrickkaleta 

From MTG we are finalizing the integration of the Bristol case study in CSIS. The completion of the data package has been possible from the developers instance, however the expert study created for Bristol is not available in this instance and it is not in production either. Is it possible that it has been definitely deleted? Is there any way to recover it?

**Comment by patrickkaleta** [comment]
> Hi @patrickkaleta
> 
> From MTG we are finalizing the integration of the Bristol case study in CSIS. The completion of the data package has been possible from the developers instance, however the expert study created for Bristol is not available in this instance and it is not in production either. Is it possible that it has been definitely deleted? Is there any way to recover it?

What can you tell me about this Study? You remember it's Study ID, the exact title or have you any URL which lead to this Study saved in your browser history?

I found two RESCCUE Studies which have the name Bristol in its titles: [166 ](https://csis-dev.myclimateservice.eu/study/166 )and [168](https://csis-dev.myclimateservice.eu/study/168). Is any one of these the one you're looking for?

If someone really removed the Study in question, than it's not possible to restore it anymore unfortunately. Also, keep in mind that **individual Studies are not synchronized** between DEV and PROD! So, if you want this Study to be visible to others, you should create it directly on PROD. Creating Studies in the DEV instance is only for you, so that you can play around with them and test out things.

**Comment by LauraMTG** [comment]
Yes, I have checked it and it is study 168, is it possible to restore it in the production instance? Thank you

**Comment by patrickkaleta** [comment]
> Yes, I have checked it and it is study 168, is it possible to restore it in the production instance? Thank you

No, restoring deleted Studies is unfortunately not possible. However, I don't think this Study was deleted on PROD. If it was created on DEV, it was always only there and never on PROD. 

Studies are not synchronized between the two environments. The synchronization process was explicitly build that way to not allow it due to possible negative side-effects on our external services (Emikat and the TM), which could get confused by the same Study being on two separate systems with possible different and contradicting settings.

**Comment by LauraMTG** [comment]
Hi @patrickkaleta 
I created a new Bristol studio in the production instance and associated the dapackage. Now you can correctly visualize the Risk and Impact resources but I can't see anything (Data/Maps) in the Hazard module. I've attached a screenshot of the Hazard Data configuration, is there anything wrong? 
![image](https://user-images.githubusercontent.com/46347135/92083893-f2bcfa00-edc6-11ea-8a75-b6eaaaa565b0.png)

**Comment by patrickkaleta** [comment]
Hi @LauraMTG, 

the screenshot you posted shows the configuration of the Study step, which is defined automatically by the chosen Study template. So, there you shouldn't need to change anything.

I believe the reason is probably that none of the resources in the DP have been associated with the Hazard Characterization step and that's why nothing is shown in that step. I'll have a closer look at this ASAP.

**Comment by patrickkaleta** [comment]
Yes, it's just what I thought. The used [data package](https://csis.myclimateservice.eu/maintenance/datapackages/5690) has in PROD has only resources that are tagged with the "Risk and Impact Assessment" EU-GL step. That's why only this step has something shown in the Data and Map tab.

I've checked the DEV version of the [data package](https://csis-dev.myclimateservice.eu/maintenance/datapackages/5194) and there it has way more resources, some of them tagged to the Hazard Characterization step and others. So, once the next synchronization is done, this data package will be updated in PROD and you will see results in the Data and Maps tab @LauraMTG

**Comment by patrickkaleta** [comment]
Currently @ghilbrae is working on a European wide Transport data package. So once that is completed, I will start the next synchronization run.

**Comment by LauraMTG** [comment]
Thank you very much @patrickkaleta 
I remain attentive to this issue

**Comment by ghilbrae** [comment]
We have created the DP https://csis-dev.myclimateservice.eu/node/5580

We may add a few more layers at some point but for now we consider it ready.
